packages/tests/test/reya_common/BaseReyaForkTest.sol

```diff
@@ -105,11 +103,8 @@ contract BaseReyaForkTest is StorageReyaForkTest {
             exchangeId: s.exchangeId
         });
 
-        s.digest = mockCoreCalculateDigest(
-            sec.core,
-            hashExecuteBySigExtended(
-                address(sec.periphery), accountId, commands, incrementedNonce, s.deadline, keccak256(abi.encode())
-            )
+        s.digest = CoreCommandHashing.mockCalculateDigest(
```

Comment:
btw, given these are fork checks is mock... an appropriate way to name a function ? (given we're working against prod and wanting to simulate realistic scenarios)

Reply:
yes, unfortunately it's hard to properly check EIP712 signatures in Solidity (from integration point of view) and this is why we need to generate them this way ... anyway I think mock prefix is not really good naming here tho (it's not mock but we somehow introduced this in core UTs and kept dragging it from there)